### PR TITLE
 JustCold

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
This PR is raised to submit for code review in the Microvers dashboard of Mohammad Rafi Amin.

Description of the task to be completed 👎

Add the HTML for the structure.
Add the CSS file for the styling.
Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
use JS structures
💯
Use Figma a collaborative UI design tool built in the browser.
Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.
This PR is raised to use GitHub pages which allows you to host (deploy) a webpage from your repository, for more information about how to use GitHub pages [Read here](https://www.w3schools.com/git/git_remote_pages.asp?remote=github) 🤩 if you want to watch a video about how to use GitHub pages instead of reading [Click here](https://www.youtube.com/watch?v=QyFcl_Fba-k&t=251s) ☺️.
This is my Capeston Project hosted on GitHub pages, so if you want to check my Capeston Project online, please 😊 [Click here](https://mrkamin.github.io/Capeston-Project-JustCold/
)
How should it be tested
Clone the repository
git clone [git@github.com](git@github.com:mrkamin/Capeston-Project-JustCold.git
Navigate to the folder

cd justcold
Checkout the branch

git checkout github_pages
Thanks for reviewing my PR 😊

### CODE REVIEW

**Comment by mrkamin** [comment]
all linter error solved

**Comment by mrkamin** [comment]
Final Commit before submit for review

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,1077 @@
...
+  --HEADER-BGCOLOR: #000;
+  --HEADER-COLOR: #fff;
+  --NAV-BGCOLOR: #fff;
+  --HERO-BGCOLOR: rgba(51, 178, 51, 0.75);
+  --HERO-COLOR: #fff;
+  --C-BRAND: #f83691;
+  --C-BRAND-RGB: 248, 54, 145;
+  --C-DARK: #303050;
+  --C-BODY: #636383;
+  --C-LIGHT: #f6f6f6;
```
Comment (medaminedev66) [inline assets/css/style.css:53]: - [ ] Following the project requirements, the pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design.
- Basic color swatch: #272A31, #EC5242, #D3D3D3
Kindly make the necessary modifications.

File: assets/js/app.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  if (document.body.scrollTop > 0 || document.documentElement.scrollTop > 0) {
+    document.getElementById('navbar').classList.add('scrolled');
+  } else {
+    document.getElementById('navbar').classList.remove('scrolled');
+  }
+}; */
+
+/* AOS.init({
+  duration: 800,
+}); */
```
Comment (medaminedev66) [inline assets/js/app.js:11]: Please, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: about.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+                  >
+                    2022
+                  </h3>
+                  <p
+                    id="sectionthree__container__overlay__p about"
+                    class="sectionthree__container__overlay__p about"
+                  >
+                    Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  </p>
+                </div>
```
Comment (medaminedev66) [inline about.html:284]: - [ ] The images in the mobile version should be arranged below one another according to the provided design. Kindly change that for a better user interface.

- Your design:
image

- Original design:
image

**Review by medaminedev66 — COMMENTED** [review]
@mrkamin?
### This is a manual review
# Request Changes ♻️ :
- [ ] Following the project requirement, you should record a video presenting the features of the project. Check out the requirements below 👇:

image


Cheers and Happy coding!:clap::clap::clap:

**Review by bellom — COMMENTED** [review]
@mrkamin 

### This is a manual review

# Approved :+1:

Cheers and Happy coding!:clap::clap::clap:

**Pull request merged**